src/main/resources/swagger/api-docs.yaml

```diff
@@ -128,6 +129,85 @@ paths:
             - profile
       x-passthrough: true
       x-passthrough-target: rawls
+  /api/billing/v2/{projectId}/spendReportConfiguration:
+    put:
+      tags:
+        - BillingV2
```

Comment:
this tag will create inconsistencies with the `billing_v2` tag but that is addressed in the other PR: https://github.com/broadinstitute/firecloud-orchestration/pull/855